tests/check/check_test.go

```diff
@@ -33,6 +37,130 @@ func TestCheckMySQL(t *testing.T) {
 	testRunAll(t, "mysql")
 }
 
+func TestCheckLogs(t *testing.T) {
+	cfg := run.MustDefaultConfigWithRandomPorts()
+	cfg.Log.Level = "none"
```

Comment:
Since you are passing a custom logger to `serverContext, `cfg.Log` will not be read within `StartServerWithContext`, right? So, this line is unnecessary..